LDM-592.tex

```diff
@@ -954,6 +954,19 @@ \section{Data Release Processing}
 In some contexts this may not remove all record of these datasets held outside their primary storage locations, as long as the storage cost of those records is negligible.
 }
 
+\usecase{DRP31}{Compare with previous data releases}{%
+
+\item
+A new data release is prepared by the Production Operators.
+
+\item
+The Validation Scientist analyzes a subset of the new data release by directly comparing it with the current Data Release.
+
+\item
+A statistical analysis is done with the combined catalogs to determine if the new data release is suitable for release.
```

Comment:
Does "combined catalogs" just mean "catalogs from both releases"?

Reply:
I think I was probably thinking some sort of cross-matched catalog from both releases but "using catalog information from both releases" probably can cover both the "catalogs from both releases" and "cross matched catalog derived from both releases".

Reply:
To the extent that this use case came from me, the intent was just that both catalogs would be available through the same Butler in a single job, not to imply that any cross-matching would already have been done.  It was meant to _facilitate_ cross-matching.